storage/innobase/trx/trx0rec.cc

```diff
@@ -1871,6 +1871,10 @@ trx_undo_report_row_operation(
 			    *clust_entry, *index, trx)) {
 			return err;
 		}
+
+		if (index->table->skip_alter_undo) {
+			return DB_SUCCESS;
+		}
 	} else {
 		bulk = false;
 	}
```

Comment:
I do not quite understand this. Why can’t we just always
```c++
		return m.first->second.bulk_insert_buffered(*clust_entry, *index, trx);
```
when we are executing a bulk insert? Why would we need to write row-level undo log records for any bulk insert?